src/Language/GraphQL/Draft/Printer/ByteString.hs

```diff
@@ -0,0 +1,34 @@
+module Language.GraphQL.Draft.Printer.ByteString where
+
+import           Data.ByteString.Builder
+import           Protolude
+
+import qualified Data.ByteString.Lazy           as BL
+import qualified Data.Text.Lazy                 as LT
+import qualified Data.Text.Lazy.Encoding        as LT
+
+import           Language.GraphQL.Draft.Printer
+import           Language.GraphQL.Draft.Syntax
+
+
+instance Printer Builder where
```

Comment:
Add inline pragmas for the functions here (and other instances). 

https://stackoverflow.com/questions/9915271/inline-pragma-in-combination-with-type-classes